backend/substrapp/utils.py

```diff
@@ -216,10 +216,12 @@ class NodeError(Exception):
 
 def get_remote_file(channel_name, url, auth, content_dst_path=None, **kwargs):
 
+    headers = {
+        'Accept': 'application/json;version=0.0',
+        'Substra-Channel-Name': channel_name
+    }
     kwargs.update({
-        'headers': {
-            'Accept': 'application/json;version=0.0',
-            'Substra-Channel-Name': channel_name},
+        'headers': {**headers, **kwargs.get('headers', {})},
```

Comment:
`headers.update(kwargs.get('headers', {}))` above and 

`'headers': headers` is easier to understand for non-python user

Reply:
fixed in https://github.com/SubstraFoundation/substra-backend/pull/395/commits/829a1bd532f6d7fbe5ffb52586899d39018da8d4